examples/thirdparty/design-antibody-antigen/design-ab-docking-w-optimization.cfg

```diff
@@ -0,0 +1,37 @@
+# ====================================================================
+#
+#  Run a docking simulation and subsequently do multiple rounds
+#   of interface optimizations targeting the HV3 loop.
+#
+# ====================================================================
+run_dir = "ab-docking-w-optimization"
+ncores = 64
+molecules =  ["data/4G6K_fv.pdb", "data/4I1B-matched.pdb"]
+# ====================================================================
+[topoaa]
+[rigidbody]
+  ambig_fname = "data/ambig.tbl"
```

Comment:
A more relevant example would be to target a specific epitope, i.e. using instead the ambiguous-CDR-NMR-CSP.tbl restraint file instead of the ambig.tbl one with targets the entire surface

Reply:
Actually you have it in another config file. But this ab initio example does not seem very relevant to me

Reply:
I took these restraints from `examples/docking-antibody-antigen/docking-antibody-antigen-CDR-accessible-clt-full-mpi.cfg` should it be another one instead?

Reply:
I would say we remove this example, there is another one doing docking with the CDR-NMR restraints, correct? That's a targeted docking to the epitope and then design. Would be the most likely scenario

Reply:
Done